# Type: test
# Subtype: T1
# English Summary: Change test server port number
# Chinese Summary: 修改测试服务器端口号
# Entry Number: 17

# === Git Diff Content ===

diff --git a/test/selenium/tests/StreamsTest.java b/test/selenium/tests/StreamsTest.java
index b3d4fb0..cf9047d 100644
--- a/test/selenium/tests/StreamsTest.java
+++ b/test/selenium/tests/StreamsTest.java
@@ -41,6 +41,7 @@ import static play.test.Helpers.*;
 public class StreamsTest extends LoggedIn {
 
     private static final int SERVER_STUB_PORT = 9005;
+    private static final int WEB_PORT = 3333;
     private ServerStub serverStub;
 
     private WebDriver driver;
@@ -77,9 +78,9 @@ public class StreamsTest extends LoggedIn {
 
     @Test
     public void addingStreamRulesWorks() {
-        running(testServer(3333), new Runnable() {
+        running(testServer(56517), new Runnable() {
             public void run() {
-                Result r = login(testBrowser(driver, 3333), serverStub, "lennart", "123123123");
+                Result r = login(testBrowser(driver), serverStub, "lennart", "123123123");
                 assertTrue("Login failed", r.isSuccess());
 
                 try {
